web/styles/left_sidebar.css

```diff
@@ -335,10 +335,6 @@ li.show-more-topics {
     #direct-messages-list {
         border-radius: 0 0 4px 4px;
     }
-
-    #more_private_messages_sidebar_title {
-        font-weight: 600;
-    }
```

Comment:
This also seems to have been introduced in 6f9e97921d39e87637c654337c77daa809080c7b and never used. Usually I'll provide this analysis from `git log -S` in the commit message, because (1) it only takes 30s to check and (2) that helps verify whether indeed the thing was unused, or it's, say, a selector constructed using JavaScript by adding `_title` to some existing code.